### PR TITLE
add numbers buttons to the UI

### PR DESCRIPTION
**Pull request by ggcarmi**

### CODE REVIEW

File: MyCalculator/ViewController.swift [hunk 1]
```diff
@@ -11,6 +11,11 @@ import UIKit
 class ViewController: UIViewController {
     
     
+    // listenet to buttons
+    @IBAction func touchDigit(_ sender: UIButton) {
+        print("touchDigit was called ")
```
Comment (oded-regev) [inline MyCalculator/ViewController.swift]: maybe there is a better way to do it?

**Review by oded-regev — COMMENTED** [review]
looks good

File: MyCalculator/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -1,21 +1,137 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="11134" systemVersion="15F34" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+<?xml version="1.0" encoding="UTF-8"?>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="11762" systemVersion="16G29" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+    <device id="retina4_7" orientation="portrait">
```
Comment (oded-regev) [inline MyCalculator/Base.lproj/Main.storyboard:3]: fdsf

**Review by oded-regev — APPROVED** [review]

**Comment by ggcarmi** [comment]
.

**Pull request merged**